packages/plugins/users-permissions/server/routes/content-api/auth.js

```diff
@@ -6,6 +6,7 @@ module.exports = [
     path: '/connect/(.*)',
     handler: 'auth.connect',
     config: {
+      auth: false,
```

Comment:
Do we really need to hard-code these as false even though the user can set permissions in users-permissions that will effectively do nothing? I think it would be better to pre-populate the database keys instead no?

Reply:
You're right, I need to dig this a little bit. Will do after the cooldown then